docs/events/winter_jam_2018.md

```diff
@@ -0,0 +1,44 @@
+<p align="center"> 
+<img src="https://cdn.discordapp.com/attachments/259454515823116290/516399623309623317/winter_jam.png">
+</p>
+
+I am happy to announce MMD's newest Jam, Winter Jam! It tastes like snow!  
+Bad jokes aside, this is a mod making event that is set to take place on the first two weeks of December.  
+Participants in this event are encouraged to create mods that fit a winter or festive theme. Festive can apply to any tradition, event, or religious ceremony that takes place during this time of year.  
+
+Unlike similar events, Winter Jam is a non competitive event.  
+The focus of this event is learning new things, challenging yourself, and encouraging the creation of new content.  
+At the end of the event, we may release a modpack to showcase the submissions. This pack may also be put on a public server, where people can play with the mods and experience them together for a limited time.  
+
+Participants and those following the event should use the [**#WinterJamMC**](https://twitter.com/search?q=%23winterjammc&src=typd) tag on twitter to talk about this event.  
+We will also be using this tag to share update, and to help us find things to retweet.  
+
+## Guidelines
+The following list of guidelines is in place to provide a fair experience for everyone. Not following these guidelines may result in your project being excluded from various rewards and/or promotions.  
+
+1. Submissions should make use of the seasonal theme of the event.  
+2. Teams should be no larger than four members.  
+3. Submissions should be made within the time frame of the event.  
+4. All submissions should be able to run on a server without crashing it. Please test on a dedicated server not a Lan server.  
+5. All assets and code should be created during the event. Open sourced assets that were made before the event are allowed.  
+6. Any open source libraries are allowed. Participants are responsible for any issues their required libraries cause.  
+7. Submissions should work with Minecraft 1.12.2, and the latest version of Forge and stable MCP for 1.12.2.  
```

Comment:
MCP mappings don't matter for compatibility, I suggest not mentioning them at all.

Reply:
```suggestion
7. Submissions should work with Minecraft 1.12.2, and the latest version of Forge for 1.12.2.  
```